### ISSUE TITLE
Speed up platesolving

### ISSUE THREAD

**Issue by keastrid** [keastrid/astroimagej#47]
- Find alternative to astronmetry.net solving
  - ansvr is Windows only
- External program interface is ok

**Comment by basilr** [comment]
I randomly found this post and thought I could help - 4yrs late. I feel a big weak point in AIJ is the time it takes to plate solve (because of the slowness of Astrometry.net, nothing to do with AIJ), but I can understand plate solving could have been  outside the scope of the original project - plate solved images were a "nice to have" but not a "need to have". That has changed for moving targets (and other features in the pipeline?), plate solved images are a "need to have", based on my experience.

I'm guessing you are aware of a program called ASTAP which does amazingly fast plate solving with a very low error rate. I believe it is pretty easy to integrate into a third party programs, and many have done that.

Other programs using ASTAP: for plate solving

CCDciel
N.I.N.A. (Nighttime Imaging 'N' Astronomy)
SharpCaap
SGP (Sequence Generator Pro)
Ekos / KStars
APT (Astro Photography Tool)

I have been using CCDciel with ASTAP for maybe 4-5years, prior to that I was using CCDciel with a local install of Astrometry.net. The difference in performance is huge, minutes vs. seconds to solve images, in almost every case.

Hope this helps.

**Comment by karenacollins** [comment]
Yeah, we're considering ASTAP as an option, but weideally desire an approach that is fully integrated with the AIJ build and installation process for simplicity to the user. We'll decide on a path soon. Thank you for you input.
